### PR TITLE
Create file.txt

### LINKED ISSUE #8: Assigned issue

### PR DESCRIPTION
**Pull request by jsjson0**
closes #8

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+New issue.
```
Comment (File5) [inline file.txt:1]: Does this is really needed?

**Review by File5 — CHANGES_REQUESTED** [review]
Add "closes" link to issue

File: file.txt (hunk 1 above)
Comment (jsjson0) [reply file.txt:1]: Yes

**Comment by jsjson0** [comment]
fixed

**Review by File5 — APPROVED** [review]

**Pull request merged**